DESCRIPTION

```diff
@@ -21,6 +21,7 @@ Depends:
     R (>= 4.1)
 Imports:
     cli,
+    dlr,
```

Comment:
I recommend putting this in Suggests. It has a small footprint so probably not a big deal, but this technically isn't required functionality. See below for some implications.